File: src/main/java/googliapparatus/entity/SongEntityStaging.java
package googliapparatus.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class SongEntityStaging {

    @Id
    private String id;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private String nameLower;

    @Column(nullable = false)
    private String link;

    @Column
    private String lyrics;

    @Column
    private String lyricsBy;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getNameLower() {
        return nameLower;
    }

    public void setNameLower(String nameLower) {
        this.nameLower = nameLower;
    }

    public void setLyrics(String lyrics) {
        this.lyrics = lyrics;
    }

    public String getLyrics() {
        return lyrics;
    }

    public String getLyricsBy() {
        return lyricsBy;
    }

    public void setLyricsBy(String lyricsBy) {
        this.lyricsBy = lyricsBy;
    }
}

File: src/main/java/googliapparatus/entity/SongEntityStaging.kt
package googliapparatus.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

@Entity
class SongEntityStaging {
    @Id
    var id: String? = null

    @Column(nullable = false)
    var name: String? = null

    @Column(nullable = false)
    var nameLower: String? = null

    @Column(nullable = false)
    var link: String? = null

    @Column
    var lyrics: String? = null

    @Column
    var lyricsBy: String? = null
}